pkgs/development/python-modules/bleach-allowlist/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, bleach
+}:
+
+buildPythonPackage rec {
+  pname = "bleach-allowlist";
+  version = "1.0.3";
+
+  src = fetchFromGitHub {
+    owner = "yourcelf";
+    repo = "bleach-allowlist";
+    # FIXME: see https://github.com/yourcelf/bleach-allowlist/issues/5
+    rev = "05484ab6a6da1ccedee25709595f11f0981b3192";
+    sha256 = "sha256-/EEw6QaCLThYZCpjtbhPF7D56DbRymkAU2DqJ2J/F8M=";
+  };
```

Comment:
If there are no tests there is little reason to fetchFromGitHub. In that case we prefer fetchPypi because it has proper versioning enforced.

Reply:
:+1: